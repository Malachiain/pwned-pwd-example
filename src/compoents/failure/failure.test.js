import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {Failure} from './index'
describe('The main app',()=>{
    it('It renders the failure page correctly',()=>{
        const wrapper = shallow(<Failure/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})