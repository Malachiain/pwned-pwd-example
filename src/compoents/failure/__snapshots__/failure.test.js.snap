// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The main app It renders the failure page correctly 1`] = `
<styled.div>
  <styled.h3>
    Something went wrong.  
  </styled.h3>
  <styled.h3>
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon="frown"
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size="2x"
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
      
  </styled.h3>
  <styled.button
    onClick={[Function]}
  >
    Try again?
  </styled.button>
</styled.div>
`;
